test/unit-test/FreeRTOS_DNS_Parser/FreeRTOS_DNS_Parser_utest.c

```diff
@@ -570,6 +611,71 @@ void test_DNS_TreatNBNS_success_nbns_non_fixed_size_buffer( void )
     ASSERT_DNS_QUERY_HOOK_CALLED();
 }
 
+/**
+ * @brief success path, formed packet is sent over the network with
+ *        vReturnEthernetFrame
+ */
+void test_DNS_TreatNBNS_success_nbns_non_fixed_size_buffer2( void )
```

Comment:
What is the difference between 1,2 and 3? Can we use better naming?

Reply:
usually the difference between 1,2,3 is to hit some conditions with a different manner for example 
if (a == 1 || b == 3) 
a != 1  || b == 3
a != 1 || b !=3
a == 1 || b